### PR TITLE
add mission statement

### PR DESCRIPTION
**Pull request by rdeits**
Adapted directly from @dehann's email, with a couple of minor typos fixed.

### CODE REVIEW

File: mission_statement.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# JuliaRobotics Mission Statement
+
+JuliaRobotics is a collection of robotics related packages that focus on robot control, simulation, navigation, and visualization.  JuliaRobotics intends to provide more visibility to robotics related work within in the Julia community.  JuliaRobotics does not intend to replace other packages in the Julia ecosystem, but rather improve the communication around combining the variety of open source packages available to open-source robotics.  JuliaRobotics intends to leverage the benefits of the Julia language.  Lastly, JuliaRobotics has a code of conduct which encourages positive, constructive collaboration, and peer review to improve access and the quality of modern robotic technology.
```
Comment (tkoolen) [inline mission_statement.md]: * robotics related -> robotics-related
* no double spaces after periods

**Review by tkoolen — COMMENTED** [review]
Thanks for doing this.

File: mission_statement.md (hunk 1 above)
Comment (rdeits) [reply mission_statement.md]: done, thanks!

**Comment by rdeits** [comment]
I'll merge this after 5pm tomorrow (March 6th) unless there are any objections.

**Comment by dehann** [comment]
Thanks!

**Pull request merged**